config/environments/production.rb

```diff
@@ -51,7 +51,7 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :debug
+  config.log_level = :warn
```

Comment:
Mainly because of the amount of messages delayed_jobs workers write which can fill your disk with completely useless logs in no time (I think it was multiple GB per day for 3 workers alone). Rails itself is not really an issue, but if there is an error or warning it just gets buried under all those debug logs. I think this is a sane default for the end-user